rfcs/0038-rosetta-construction-api.md

```diff
@@ -407,7 +447,7 @@ Luckily Rosetta has a very clear specification, so our designs are mostly constr
 
 In [marshal keys (c)](#marshalkeys), we could also change commands that accept public keys to also accept this new format. Additionally, we could change the GraphQL API to support this new format too. I think both of these changes are unnecessary to prioritize as the normal flows will still be fine and we'll still encourage folks to pass around the standard base58-encoded compressed public keys as they are shorter.
 
-In the sections about [encoding unsigned transactions](#encoded-unsigned-transaction) and [encoding signed transactions](#encoded-signed-transaction), we make an explicit decision to pick a format that fits what our client-sdk expects. This was done to improve implementation velocity and because we did conciously choose that interface with usability in mind. Additionaly, using a readable JSON string makes it easy to audit, debug, and understand our implementation. This design does subsequently require extra processing for other signers (like a ledger hardware device or a native binary).
+In the sections about [encoding unsigned transactions](#encoded-unsigned-transaction) and [encoding signed transactions](#encoded-signed-transaction), we make an explicit decision to pick a format that supports arbitrary signers. There is minimal change involved with the client-sdk to make that supported; additionally, this was done to improve implementation velocity and because we did conciously choose that interface with usability in mind. JSON is chosen to pack products of data as using a readable JSON string makes it easy to audit, debug, and understand our implementation.
```

Comment:
"pack products" is talking about the data sums issue? not clear to me

Reply:
just the tuples of eg signature, payment, stakeDelegation. Rather than encoding them with offsets or jump tables or something like that.